CoffeeChat/Controller/Endpoints.swift

```diff
@@ -24,36 +24,70 @@ extension Endpoint {
         Endpoint.config.host = baseURL
         Endpoint.config.commonPath = "/api/v1"
     }
+    
+    static var standardHeaders: [String: String] {
+        if let token = UserDefaults.standard.string(forKey: Constants.UserDefaults.accessToken) {
+            return ["Authorization": "Bearer \(token)"]
+        } else {
+            return [:]
+        }
+    }
+
+    static var updateHeaders: [String: String] {
+        if let token = UserDefaults.standard.string(forKey: Constants.UserDefaults.refreshToken) {
+            return ["Authorization": "Bearer \(token)"]
+        } else {
+            return [:]
+        }
+    }
 
     /// [GET] Check if server application is running
     static func pingServer() -> Endpoint {
         Endpoint(path: "/general/hello/")
     }
+    
+    static func refreshUserToken(token: String) -> Endpoint {
+        return Endpoint(path: "/refresh/", headers: updateHeaders)
+    }
 
     /// [POST] Authenticate ID token from Google and creates a user if account does not exist
     static func createUser(idToken: String) -> Endpoint {
         let body = UserSessionBody(idToken: idToken)
         return Endpoint(path: "/auth/login/", body: body)
     }
 
-    /// [POST] Update information about the user
-    static func updateUser(
-        clubs: [String],
+    /// [POST] Update demographics information about the user
+    static func updateUserDemographics(
         graduationYear: String,
         hometown: String,
-        interests: [String],
         major: String,
-        pronouns: String
+        pronouns: String,
+        profilePictureURL: String
     ) -> Endpoint {
-        let body = UserUpdateBody(
-            clubs: clubs,
+        let body = UserUpdateDemographicsBody(
```

Comment:
Would creating a`UserDemographics` model (wouldn't you already have this from the user too?), or something of the sorts, be helpful so we only need to pass in one param? 5 is on the higher end...